### PR TITLE
Fixing Voiceflow API Blocks.

### PR DESCRIPTION
**Pull request by molplu**
Changes Martin made to fix the block creation on Voiceflow.

### CODE REVIEW

File: transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/BlockRequest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public BlockRequest(String emailAddress, String password, String diagramID,
+                        String intent1, String intent2, String intent3) {
+        this.emailAddress = emailAddress;
+        this.password = password;
+        this.diagramID = diagramID;
+        this.intent1 = intent1;
+        this.intent2 = intent2;
+        this.intent3 = intent3;
+    }
+
```
Comment (KareemSalem7) [inline transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/BlockRequest.java:23]: You could add a comment here basically saying these are getter functions

File: transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java [hunk 2]
```diff
@@ -56,6 +65,34 @@ public ArrayList<ArrayList<ArrayList<String>>>  getTranscriptData_2(@RequestBody
         return interactor.getTranscriptData_2(UserApi);
     }
 
+    @GetMapping("/threeIntents/api") // Returns the top three intents to be added to the front end
+    public ArrayList<ArrayList<String>> getTreeThreeIntentsWithAPI
+            (@RequestHeader("Authorization") String authHeader) throws IOException {
+
+
```
Comment (KareemSalem7) [inline transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java:72]: Remove these print statements when your done debugging

File: transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java [hunk 3]
```diff
@@ -35,11 +41,14 @@ public void storeUserInfo(@RequestBody UserInfo user) {
...
-        String password = account.getPassword();
-        String diagramID = account.getDiagramID();
+    public void createVoiceflowBlock(@RequestBody BlockRequest blockRequest) throws Exception {
+        String email = blockRequest.getEmailAddress();
+        String password = blockRequest.getPassword();
+        String diagramID = blockRequest.getDiagramID();
+        String intent1 = blockRequest.getIntent1();
+        String intent2 = blockRequest.getIntent2();
+        String intent3 = blockRequest.getIntent3();
+        System.out.println(intent1);
```
Comment (KareemSalem7) [inline transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java:51]: remove print statements

**Review by KareemSalem7 — APPROVED** [review]
remove the old print statements and include some comments

File: transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java [hunk 4]
```diff
@@ -35,11 +41,14 @@ public void storeUserInfo(@RequestBody UserInfo user) {
...
 
     @PostMapping("/createBlock")
-    public void createVoiceflowBlock(@RequestBody Account account,
-                                     String intent1, String intent2, String intent3) throws Exception {
-        String email = account.getEmailAddress();
-        String password = account.getPassword();
-        String diagramID = account.getDiagramID();
+    public void createVoiceflowBlock(@RequestBody BlockRequest blockRequest) throws Exception {
+        String email = blockRequest.getEmailAddress();
+        String password = blockRequest.getPassword();
```
Comment (CharlieMorocz) [inline transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestController.java:46]: May want to add some comments to clarify what this does.

**Review by CharlieMorocz — APPROVED** [review]
Looks good, just make sure to add some documentation!

**Pull request merged**